templates/react/entry-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/Create{{pascal_case entry_type.name}}.tsx.hbs

```diff
@@ -0,0 +1,88 @@
+import type { AppClient, Record, EntryHash, AgentPubKey, ActionHash, DnaHash } from '@holochain/client';
+import { FC, useState, useContext, useEffect } from 'react';
+
+import type { {{pascal_case entry_type.name}}{{#each entry_type.fields}}{{#if (eq field_type.type "Enum")}}, {{field_type.label}}{{/if}}{{/each}} } from './types';
+import { HolochainContext } from '../../contexts/HolochainContext';
+{{#uniq_lines}}
+  {{#each entry_type.fields}}
+    {{#if widget}}
+{{> (concat field_type.type "/" widget "/edit/imports") }}
+
+    {{/if}}
+  {{/each}}
+{{/uniq_lines}}
+
+const Create{{pascal_case entry_type.name}}: FC<Create{{pascal_case entry_type.name}}Props> = ({ on{{pascal_case entry_type.name}}Created{{#each entry_type.fields}}{{#if (not widget) }}{{#if (eq cardinality "vector")}}, {{camel_case field_name}}{{else}}, {{camel_case field_name}}{{/if}}{{/if}}{{/each}} }) => {
+  const {client} = useContext(HolochainContext);
+{{#each entry_type.fields}}
+  {{#if widget }}
+    {{#if (not (eq cardinality "vector" ) )}}
+	const [{{camel_case field_name}}, set{{pascal_case field_name}}] = useState<{{> (concat field_type.type "/type")}}>({{> (concat field_type.type "/sample")}});
+    {{else}}
+	const [{{camel_case field_name}}, set{{pascal_case field_name}}] = useState<{{> (concat field_type.type "/type")}}[]>({{> (concat field_type.type "/sample")}});
```

Comment:
```suggestion
	const [{{camel_case field_name}}, set{{pascal_case field_name}}] = useState<{{> (concat field_type.type "/type")}}[]>([{{> (concat field_type.type "/sample")}}]);
```

This gets rid of an error message that causes the form to not load... However, even with this fix, the widget (in my case a widget for `Vec<String>`) doesn't show any items. No visible errors; not sure how to debug this.

Reply:
Updated this to dynamically set the type as an Array<field_type> or field_type depending on `cardinality`